### DISCUSSION TITLE
Unknown command:"NEOPIXEL_ON"

### DISCUSSION THREAD

**Discussion by dannler** [Guilouz/Klipper-Flsun-Speeder-Pad#107] [Q&A] [answered]
im getting this error everytime after i completed a print, i did not install neopixel, so why would it want to give this command?

**Comment by fzechert** [comment]
I can confirm this problem on my printer. I think it started occurring after the last update of the printer?!

I searched all my config files, and I cannot find `NEOPIXEL_ON` in any configuration file. So I do not know where this is invoked from.

It happens on the Print-End-Screen. Every time I press the button "Main Menu" an error is logged in the console `Unknown command:"NEOPIXEL_ON"`. Sometimes the button still brings me back to the main menu, sometimes I am completely stuck on the print-end-screen.

If anybody knows of a solution I would be glad to hear about it.

**Comment by Guilouz** [comment, marked as answer]
This command is hard coded in KlipperScreen to reload neopixels to white color when back to main menu after a finish print (neopixels in green color).

I will add a fix to load this command only when neopixels are used.

**Comment by fzechert** [comment]
In the meantime, you can add the following code to macros.cfg to suppress the error message:

```ini
# ~~~~
# Dirty fix for NEOPIXEL_ON
# ~~~~
[gcode_macro NEOPIXEL_ON]
gcode: 
; NOOP
```

**Reply by Guilouz** [reply]
I have just fixed this, please update KlipperScreen to latest version.

**Reply by fzechert** [reply]
I updated and did a test print. I can confirm the problem is fixed. Thank you very much for the super fast fix & update. :tada:
